examples/database_components/src/lib.rs

```diff
@@ -46,7 +72,7 @@ impl<S: State, BH: BlockHash> Database for DatabaseComponents<S, BH> {
     }
 
     fn block_hash(&mut self, number: u64) -> Result<B256, Self::Error> {
-        self.block_hash
+        WrapBlockHashRef(&self.block_hash)
```

Comment:
Why do we need `Wrap..` here can't we use `self.block_hash` directly?

Reply:
tbh, i got caught up in following the pattern from the rest of the lib. you;re right this is unnecessary